<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationWidget</class>
 <widget class="QWidget" name="LocationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>683</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="airportPage">
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QRadioButton" name="parkingRadio">
         <property name="text">
          <string>Parking:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="runwayRadio">
         <property name="text">
          <string>Runway:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="onFinalCheckbox">
         <property name="text">
          <string>On final approach at distance:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QComboBox" name="parkingCombo"/>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QComboBox" name="runwayCombo"/>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="AirportDiagram" name="airportDiagram" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="approachDistanceSpin">
         <property name="suffix">
          <string>nm</string>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="navaidPage">
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="1" colspan="2">
        <widget class="QSpinBox" name="airspeedSpinbox">
         <property name="suffix">
          <string>kts</string>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>120</number>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QSpinBox" name="altitudeSpinbox">
         <property name="suffix">
          <string>ft</string>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>120000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="altitudeLabel">
         <property name="text">
          <string>Altitude:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2" colspan="8">
        <widget class="NavaidDiagram" name="navaidDiagram" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="6" colspan="2">
        <widget class="QComboBox" name="altitudeModeCombo">
         <item>
          <property name="text">
           <string>Barometric altitude (ASL)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Above ground (AGL)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Flight Level (FL)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aispeedLabel">
         <property name="text">
          <string>Airspeed:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="8">
        <widget class="QGroupBox" name="offsetGroup">
         <property name="title">
          <string>Offset</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="offsetBearingLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Bearing:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="offsetBearingSpinbox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="trueBearing">
            <property name="text">
             <string>True</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="offsetDistanceLabel">
            <property name="text">
             <string>Distance:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="offsetNmSpinbox">
            <property name="suffix">
             <string>nm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="locationSearchEdit">
           <property name="placeholderText">
            <string>Enter an ICAO code, navaid or search by name</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchHistory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="searchStatusText">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="searchIcon">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="searchResultsList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AirportDiagram</class>
   <extends>QWidget</extends>
   <header location="global">GUI/AirportDiagram.hxx</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NavaidDiagram</class>
   <extends>QWidget</extends>
   <header location="global">GUI/NavaidDiagram.hxx</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
